includes/rest-api/CMB2_REST.php

```diff
@@ -117,6 +124,10 @@ public static function maybe_init_and_hookup( CMB2 $cmb ) {
 	public function __construct( CMB2 $cmb ) {
 		$this->cmb = $cmb;
 		self::$boxes[ $cmb->cmb_id ] = $this;
+		self::$rest_slug = 'cmb2';
+		if( !empty( $this->cmb->prop( 'rest_slug_group' ) ) ) {
```

Comment:
@Mte90 Please improve code style - there should be a space after `!`: `! empty()`.

Reply:
Also, you can't use a an evaluated expression in `empty` in earlier versions of PHP. E.g. the value inside `empty()` needs to be a variable.